geps/gep-1619.md

```diff
@@ -62,24 +73,160 @@ Header-based stateful sessions are achieved by a backend or gateway providing an
 
 Session information can be also encoded into the request URL to establish a persistent session. The server rewrites the client's URL to encode the new session information automatically. The server then decodes the session information from the URL to identify the session.
```

Comment:
"The server then decodes the session information from the URL to identify the session."

(The server **or** the Gateway ?)